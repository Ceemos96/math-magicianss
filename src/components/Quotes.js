import React from 'react';

const Quotes = () => (
  <div className="quotes-wrapper">
    <h2>Quote of the day</h2>
    <p>
      There are two ways to do great mathematics. The first is to be smarter than
      everybody else. The second way is to be stupider than everybody else â€” but persistent.
    </p>
  </div>
);
export default Quotes;
