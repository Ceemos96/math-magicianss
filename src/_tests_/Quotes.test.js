import React from 'react';
import renderer from 'react-test-renderer';
import Quotes from '../components/Quotes';

describe('Testing the Quotes function', () => {
  it('Home is rendered', () => {
    const quotes = renderer
      .create(<Quotes />)
      .toJSON();
    expect(quotes).toMatchSnapshot();
  });
});
