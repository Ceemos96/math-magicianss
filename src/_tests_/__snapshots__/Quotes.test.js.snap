// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing the Quotes function Home is rendered 1`] = `
<div
  className="quotes-wrapper"
>
  <h2>
    Quote of the day
  </h2>
  <p>
    There are two ways to do great mathematics. The first is to be smarter than everybody else. The second way is to be stupider than everybody else â€” but persistent.
  </p>
</div>
`;
